packages/react-components/src/components/Banner/Banner.jsx

```diff
@@ -137,7 +137,7 @@ export class Banner extends Component {
           visible
           full-width
           closeable={showClose}
-          onClose={onCloseAlert}
+          onVaClose={onCloseAlert}
```

Comment:
I'm wondering if we need a more descriptive name here. Would something like `onCloseEvent` be clearer?

Reply:
Most of our custom events have been prefixed with `va`. If we're okay moving away from that, I think that's a much better name. If we want a compromise, how about `onVaCloseEvent`?

Reply:
Either of those is fine - those `va*` event prefixes were originally a workaround to get rid of some stencil build warnings for custom names conflicting with native names. We may want to give those other events some attention at some point :sweat_smile:

Reply:
yeah, I'm not too keen on the `va*` prefixes, it's not clear on first glance what's going on there.

Reply:
Sounds good to me. I've updated it to `closeEvent` and used as `onCloseEvent`.